README.md

```diff
@@ -0,0 +1,44 @@
+# Data Engineering Internship Repository
```

Comment:
Please substitute `DBT` with `dbt`. This was a specific creators choice to name it in lower case.
This is not about [this](https://en.wikipedia.org/wiki/Dialectical_behavior_therapy)

Reply:
Please change it everywhere `DBT` should be replaced with `dbt`